src/compas_rhino/artists/artist.py

```diff
@@ -55,7 +59,7 @@ class Artist(object):
 
     __module__ = "compas_rhino.artists"
 
-    def __init__(self, layer=None):
+    def __init__(self, geometry=None, attributes=None, layer=None):
```

Comment:
maybe `layer` is an attribute too?

Reply:
agreed. everything that is relevant for serialisation should go in attributes.
but i would add the option of `**kwargs` that allows for overwriting some of the values in the attribute dict using name-value pairs. same as for `add_vertex`, for example...

Reply:
yep was thinking the same, let me see if there is a way to adjust this without breaking the old api